worlds/civ_6/Options.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass
+from Options import Choice, DeathLink, DefaultOnToggle, PerGameCommonOptions, Range, Toggle
+
+
+class ProgressionStyle(Choice):
+    """Determines what progressive items (if any) should be included.\n\n
+    Districts Only: Each tech/civic that would normally unlock a district or district building now has a logical progression. Example: TECH_BRONZE_WORKING is now PROGRESSIVE_ENCAMPMENT\n\n
+    Eras and Districts: Players will be defeated if they play until the world era advances beyond the currently unlocked maximum era.
+    A notification will be shown as the end of the era approaches letting the player know if they don't have enough progressive era items.
+    Unlocked eras can be seen in both the tech and civic trees. Includes all progressive districts.\n\n
+    None: No progressive items will be included. This means you can get district upgrades that won't be usable until the relevant district is unlocked.
+    """
+    display_name = "Progression Style"
+    option_districts_only = 0
+    option_eras_and_districts = 1
+    option_none = 2
+    default = option_districts_only
+
+
+class ShuffleGoodyHuts(DefaultOnToggle):
+    """Shuffles the goody hut rewards. Goody huts will only contain junk items and location checks are received sequentially (GOODY_HUT_1, GOODY_HUT_2, etc)."""
+    display_name = "Shuffle Goody Hut Rewards"
+    default = True
+
+
+class BoostSanity(Toggle):
+    """Boosts for Civics/Techs are location checks. Boosts can now be triggered even if the item has already been researched. If it is dependent upon a unit that is now obsolete, you can click toggle on/off the relevant tech in the tech tree."""
+    default = False
```

Comment:
You have display_name on some of these but not others. I recommend loading up the webhost locally and making sure the display names for each of these on the options page look the way you want them to.

And also you should check that the option descriptions display the way you want them to on the webhost.

Reply:
Here's what it looks like right now, for reference:

![image](https://github.com/user-attachments/assets/7be164cb-8a44-4871-ad85-6c3d2a6e66b8)

I agree with adding `display_name` so they look consistent

Reply:
Ah definitely spaced it on the most recent ones. Thank you! (And thanks for grabbing the screenshot for me 🔥 )